index.bs

```diff
@@ -2478,8 +2478,11 @@ To <dfn>process an attribution source</dfn> given an [=attribution source=] |sou
 1. If |destinationRateLimitResult| is "<code>[=destination rate-limit result/hit reporting limit=]</code>":
     1. Run [=obtain and deliver a debug report on source registration=] with "<code>[=source debug data type/source-destination-rate-limit=]</code>" and |source|.
     1. Return.
+1. Let |debugDataType| be "<code>[=source debug data type/source-success=]</code>".
+1. Set |debugDataType| to "<code>[=source debug data type/source-noised=]</code>"
+    if |source|'s [=attribution source/randomized response=] is not null.
```

Comment:
Did you mean, "is null," rather than "is not null?"

Reply:
The source is noised if the response "is not null" right?

Reply:
Right. I thought the intention if "hit global limit" is to send a source-noised debug report even if the source was not noised.

Reply:
The intention is actually a "source-noised" debug report would be sent no matter whether the global limit is hit or not. It should be just based on the response config.

Reply:
Yes, that's the intention. The same debug report should be sent no matter global limit hit or not.

Reply:
Gotcha